src/cpp/rtps/participant/RTPSParticipantImpl.cpp

```diff
@@ -1553,6 +1556,8 @@ bool RTPSParticipantImpl::get_persistence_service(
         {
             logError(RTPS_PARTICIPANT,
                     "Couldn't create writer persistence service for transient/persistent " << debug_label);
+            // Error log level can be disable. Avoid unused warning
+            (void) debug_label;
```

Comment:
Remove `debug_label` parameter and correct log accordingly. Currently, the parameter is used when calling the function to log either reader or writer. This information is included in the `EndpointAttributes`